### ISSUE TITLE
Update `appmap_dir` of existing `appmap.yml` files when launching CLI tools

### ISSUE THREAD

**Issue by jansorg** [applandinc/appmap-intellij-plugin#311]
It should use `appmap_dir: tmp/appmap` unless it's a Java project where we can find a better default (e.g. `build/appmap`).
This should happen before an AppMap CLI tool is launched.

**Comment by kgilpin** [comment]
@brikelly @apotterri are we going with `tmp/appmap` everywhere now?

**Comment by brikelly** [comment]
Yes, @apotterri has the changes to the agent, Gradle plugin, and Maven plugin ready, and I've got the doc changes ready for the move to `tmp/appmap` as well. We just need the IntelliJ plugin changed as well before we can release that work, which is in @jansorg 's queue.

**Comment by kgilpin** [comment]
@jansorg is there any case now where we are generating an appmap directory other than appmap_dir?
